src/ERC4626UpgradeableSafe.sol

```diff
@@ -8,13 +8,22 @@ import {ERC4626Upgradeable, ERC20Upgradeable, IERC20MetadataUpgradeable} from "@
 /// @custom:contact security@morpho.xyz
 /// @notice ERC4626 Tokenized Vault abstract upgradeable implementation tweaking OZ's implementation to make it safer at initialization.
 abstract contract ERC4626UpgradeableSafe is ERC4626Upgradeable {
+    /// STORAGE ///
+
+    /// @dev This empty reserved space is put in place to allow future versions to add new
+    /// variables without shifting down storage in the inheritance chain.
+    /// See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+    uint256[50] private __gap;
+
     /// CONSTRUCTOR ///
 
     /// @notice Constructs the contract.
     /// @dev The contract is automatically marked as initialized when deployed so that nobody can highjack the implementation contract.
```

Comment:
```suggestion
    /// @dev The contract automatically disables initializers when deployed so that nobody can highjack the implementation contract.
```